R/read_alevin.R

```diff
@@ -169,18 +179,24 @@ read_tximport <- function(quant_dir) {
 #'
 #' @param quant_dir Path alevin output directory.
 #' @param tech_version Technology or kit used to process library (i.e. 10Xv3, 10Xv3.1).
+#' @param assay_ontology_term_id Optional Experimental Factor Ontology term associated with the provided `tech_version`
+#' @param seq_unit Optional sequencing unit associated with the library (i.e. cell, nucleus)
 #' @param library_id Optional library identifier
 #' @param sample_id Optional sample identifier.
 #'   If multiplexed samples are included in a library, this may be a vector.
+#' @param project_id Optional project identifier.
 #'
 #' @return A list containing alevin run metadata,
 #'   with NULL values for missing elements.
 #'
 #' @noRd
 read_alevin_metadata <- function(quant_dir,
                                  tech_version,
+                                 assay_ontology_term_id = NULL,
+                                 seq_unit = NULL,
```

Comment:
Before merging, can we be more consistent about argument order between these functions?

Reply:
Sorry I can reopen it... I was just following the order that was there and grouping the seq unit/ tech/ ontology id together and then the library id/ sample id/ project id together.

Reply:
I guess I'm too late.